### PR TITLE
Addresses Issue 80: Adding travel information

### PR DESCRIPTION
**Pull request by SamFritz**
This PR addresses [issue 80](https://github.com/archivesunleashed/archivesunleashed.org/issues/80) and adds in travel information for GW participants to the website including info about: 
  accommodation
  airport
  transportation options to/from airports
  local transportation options
  local attractions

Asking for review by @ianmilligan1 to check if any important information has been overlooked, especially when it comes to transport options. 

Thanks!!

### CODE REVIEW

File: content/washington/index.md [hunk 1]
```diff
@@ -68,9 +61,69 @@ George Washington University - Foggy Bottom Campus
 
 <iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d12420.328684893137!2d-77.0483541!3d38.8992362!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0xf0d1febd782fab15!2sGelman+Library!5e0!3m2!1sen!2sca!4v1541111558124" width="600" height="400" frameborder="0" style="border:0" allowfullscreen></iframe>
 
+## Travel to Washington
+
+The AUT team has put together some information and resources to help with your travel plans.
```
Comment (ianmilligan1) [inline content/washington/index.md]: "AUT team" -> "Archives Unleashed team"

File: content/washington/index.md [hunk 2]
```diff
@@ -68,9 +61,69 @@ George Washington University - Foggy Bottom Campus
...
+
+![Accommodations](/images/accommodations.png)
+
+There are many choices for accommodations in the DC area! We encourage all of our out-of-town participants to check out the [DC Metro Rail Map](https://www.wmata.com/schedules/maps/upload/2017-System-Map.pdf) to help spot areas close to campus.
```
Comment (ianmilligan1) [inline content/washington/index.md]: "to help spot areas close to campus. The Gelman Library is located near the Foggy Bottom-GWU station."

File: content/washington/index.md [hunk 3]
```diff
@@ -68,9 +61,69 @@ George Washington University - Foggy Bottom Campus
...
+| Taxi                      | [Taxis](http://www.flyreagan.com/dca/taxi-service) are available outside Terminals A, B, and C and are available on a first-come, first-serve basis. All taxicabs accept cash or credit cards and [metered rates](http://www.flyreagan.com/sites/default/files/taxi_rate_dca_2016.pdf) apply.                           | [Washington Flyer Taxicabs](http://www.flydulles.com/iad/washington-flyer-taxi-service) serve the Dulles airport exclusively, with 24-hour service. To find taxis, follow the Ground Transportation signs down the ramp to Door 2 and Door 6. All taxicabs accept cash or credit cards. Please confirm with the Airport Taxi Counter for taxi rates, estimates for fares, as of 2017, can be found [here](http://www.flydulles.com/iad/estimated-taxi-fares). |
```
Comment (ianmilligan1) [inline content/washington/index.md]: Maybe also note that Uber and Lyft are options as well.

File: content/washington/index.md [hunk 4]
```diff
@@ -68,9 +61,69 @@ George Washington University - Foggy Bottom Campus
...
+| Car Rentals               | Located inside the airport, there are several companies that offer [car rental services](http://www.flyreagan.com/dca/rental-cars).                                                                                                                                                                                     | You can **[rent a car](http://www.flydulles.com/iad/rental-cars)** at Dulles from several companies, with complimentary shuttles between the rental lot and the terminal.                                                                                                                                                                                                                                                                                     |
```
Comment (ianmilligan1) [inline content/washington/index.md]: Let's maybe remove car rentals... on the off chance this lets people think that we'd like to deal with reimbursing car rental fees for intra-DC travel. 😄

File: content/washington/index.md [hunk 5]
```diff
@@ -68,9 +61,69 @@ George Washington University - Foggy Bottom Campus
...
+![Downtown Transit](/images/transit.png)
+
+While you are in Washington there are a few ways to get around, but ultimately it will depend on how close you are to the University. The Gelman Library is located on Foggy Bottom campus and is easily accessible by public transportation or by foot from most areas of downtown Washington.
+
+* **Public Transport** - Metrorail provides access to public transportation in Washington. They have a great [trip planner](https://www.wmata.com/schedules/trip-planner/). Depending on whether you will be doing any extra traveling around the city, you may want to consider a [SmarTrip® card](https://www.wmata.com/fares/smartrip/) for multi-ride usage.
+
+>**Notes**:
+- Metrorail fare for regular routes is $2.00USD using a SmarTrip® card or cash, and for express routes is $4.25USD. Fares cards can be purchased at [authorized outlets](https://www.wmata.com/fares/basic.cfm) including: vending machines at Metrorail stations, online, Metro sales office, and specific commuter stores and retail outlets.
+
+* **Taxis** - a few options for getting around via taxi include:
```
Comment (ianmilligan1) [inline content/washington/index.md]: Note rideshare apps like Uber and Lyft

File: content/washington/index.md [hunk 6]
```diff
@@ -68,9 +61,69 @@ George Washington University - Foggy Bottom Campus
...
+  - [Red Cab DC](http://www.redcabdc.com) - 202.285.1632
+  - [Taxi Transportation Service](http://dctaxionline.com/) - 202.398.0500
+
+![Attractions](/images/attractions.png)
+
+For those adventurers who’d like to check out Washington attractions, you may want to check out the [Visitor's Guide to Washington, DC](https://washington.org/things-do-washington-dc). Some of our favourite spots to visit include:
+
+* [Library of Congress](https://www.loc.gov) - why not stop by the Jefferson building for a free one-hour walking [tour](https://www.loc.gov/visit/)?
+* [National Archives](https://www.archives.gov/dc) - did you know you can make a reservation to tour the National Archives? This means skipping the General Admission line and heading straight in!
+* [Georgetown](https://washington.org/visit-dc/things-see-do-georgetown-washington-dc) - it's a little further out from GWU, but this is a great area for exploring, shopping, and the restaurants are incredible!
```
Comment (ianmilligan1) [inline content/washington/index.md]: Not too far, actually, probably only like a 20-30 minute walk west? If my memory works properly..

File: content/washington/index.md [hunk 7]
```diff
@@ -68,9 +61,69 @@ George Washington University - Foggy Bottom Campus
...
+* [Smithsonian](https://www.si.edu/museums) - offering 19 museums, galleries, gardens, and a zoo, visitors will have plenty to explore. Check out the complete list of [museums](https://www.si.edu/museums) to start planning your visit.
```
Comment (ianmilligan1) [inline content/washington/index.md]: Smithsonian -> Smithsonian museums

File: content/washington/index.md [hunk 8]
```diff
@@ -68,9 +61,69 @@ George Washington University - Foggy Bottom Campus
...
+* [DuPont Circle](https://washington.org/topics/dupont-circle) - another great neighbourhood to visit, with bistros, bookstores, unique restaurants, and lively entertainment.
```
Comment (ianmilligan1) [inline content/washington/index.md]: Just Dupont (not DuPont)

File: content/washington/index.md (hunk 3 above)
Comment (SamFritz) [reply content/washington/index.md]: I can add Uber and Lyft to local transport (getting around town) but looks Dulles has some pretty strict restrictions around non-Washington flyer taxis/transport that pick up passengers.
You definitely need to call ahead for pick getting into town - no soliciting at airport.

Comment (ianmilligan1) [reply content/washington/index.md]: Yep, I've used Uber and Lyft out of DC airports - the app tells you where to go.

File: content/washington/index.md (hunk 7 above)
Comment (SamFritz) [reply content/washington/index.md]: Thanks, wasn't sure if it was just Smithsonian, Smithsonian Institute, etc.

**Pull request merged**